src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -93,6 +116,66 @@ PluginResponse handlePluginsRequest(PluginRequest pluginRequest) {
         return pluginResponse;
     }
 
+    NamedWriteableRegistryResponse handleNamedWriteableRegistryRequest(NamedWriteableRegistryRequest request) {
```

Comment:
Its nice to see how we are adding more extension points support for the SDK. 
This is expanding ExtensionsRunner. 

Instead could you write a new class (may be `NamedWriteableRegistryApi`) which is constructed during the bootstrap of extension, populate `namedWritables` and write up functions which handle the request, parseRequest etc

Reply:
Great suggestion, I'll definitely add this change in my next commit